GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitCommandsTests.cs

```diff
@@ -1036,10 +1036,10 @@ public void RenameOnlyFileInFolder()
         public void UpdateIndexCannotModifySkipWorktreeBit()

```

Comment:
This seems like a duplicate test to the ones you added?  If so I would just delete it.